council/council-steps.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8" >
+<title>Formal Objections: From Objection to W3C Council</title>
+<link rel="stylesheet" href="/StyleSheets/generic-base-1.css" type="text/css">
+<link rel="stylesheet" type="text/css" href="../assets/main.css">
+<link rel="shortcut icon" href="/Icons/WWW/Literature.gif">
+<link rel="start" href="../" >
+<style>
+  .IssueLabel-big {
+    font-weight: 600;
+    padding: 4px 8px;
+    border-radius: 3px;
+    line-height: 1.5;
+    box-sizing: border-box;
+    text-decoration: none;
+  }
+</style>
+</head>
+<body>
+
+<div id="header">
+<span class="logo"><a href="/"><img src="/Icons/WWW/w3c_home_nb"
+alt="W3C" height="48" width="72"></a></span>
+    <div class="breadcrumb">
+    <a href="/Member/">Member</a> &#x2192; <a href="/Guide/">The Art of
+Consensus</a> &#x2192;
+<h1>Formal Objections: From Objection to W3C Council</h1>
```

Comment:
meta comment: since we're adding documentation about Team internal procedures into /Guide (like TilT), we should probably make those pages  are not expected to be followed by non-Team folks.